### PR TITLE
Use mongo instead of postgres for fight service

### LINKED ISSUE #6: Introduce a different database for each microservice ?
All three Villain, Hero and Fight microservices use a Postgres database. What about having a different database type for each microservice?

We could use Postgres, MySQL and MariaDB, for example.

### PR DESCRIPTION
**Pull request by edeandrea**
Use mongo instead of postgres for the fights service.

Closes #6

### CODE REVIEW

**Comment by edeandrea** [comment]
@cescoffier / @agoncal Would you mind taking a look at this? This changes the fight service DB from postgres to Mongo (as in #6)

File: event-statistics/src/main/avro/fight.avsc [hunk 1]
```diff
@@ -6,8 +6,8 @@
   "fields": [
     {
       "name": "id",
-      "type": "long",
-      "default": -1
+      "type": "string",
+      "default": ""
```
Comment (cescoffier) [inline event-statistics/src/main/avro/fight.avsc:10]: shouldn't we use `null`?

**Review by cescoffier — APPROVED** [review]
LGTM.

File: event-statistics/src/main/avro/fight.avsc (hunk 1 above)
Comment (edeandrea) [reply event-statistics/src/main/avro/fight.avsc:10]: Actually I'll do one better. It wasn't until just now that I understood how `null` works in Avro :) I'll push a new commit in a minute or 2.

Comment (edeandrea) [reply event-statistics/src/main/avro/fight.avsc]: @cescoffier I pushed a new commit.

**Pull request merged**